app/assets/javascripts/oxalis/view/right-menu/dataset_info_tab_view.js

```diff
@@ -84,12 +84,11 @@ const shortcuts = [
 ];
 
 export function calculateZoomLevel(flycam: Flycam, dataset: APIDataset): number {
-  const zoom = getPlaneScalingFactor(flycam);
+  // todo
+  const zoom = getZoomValue(flycam);
   let width;
   const { viewMode } = Store.getState().temporaryConfiguration;
-  if (constants.MODES_PLANE.includes(viewMode)) {
-    width = constants.PLANE_WIDTH;
-  } else if (constants.MODES_ARBITRARY.includes(viewMode)) {
+  if (constants.MODES_PLANE.includes(viewMode) || constants.MODES_ARBITRARY.includes(viewMode)) {
```

Comment:
We can probably get rid of this whole if/else and simply replace it with `width = constants.VIEWPORT_WIDTH;`